elastic/kibana
# [ResponseOps][Flapping] detect flapping start / end, set flapping state

Resolves https://github.com/elastic/kibana/issues/143443

## Summary

Added processing to track flapping state, and detect when an alert is flapping.
When an alert is determined to be flapping we will log a message for now.


### Checklist


### To verify
- Create a rule that will change between active and recovered. I like to create a rule like `index threshold` where I can force the state to change.
- Look at the task manager document for this rule and verify that the `flappingHistory` in the `state` is being updated properly.
- Let the rule run and verify that if the alert is flapping in AAD, Event Log and alert summary.
